sbtplugin/src/main/scala/com/typesafe/tools/mima/plugin/SbtMima.scala

```diff
@@ -129,16 +129,21 @@ object SbtMima {
           }
         }.transform(Success(_), ex => Failure(new ParsingException(file, line, ex)))
 
+      val lines = try {
+        Source.fromFile(file).getLines().toVector
+      } catch {
+        case t: Throwable => throw new RuntimeException(s"Couldn't load '$file'")
```

Comment:
Could you add the original `Throwable` as the cause (to avoid swallowing the stack)?

Also, can we be more selective on what throwables to catch?

Reply:
What about those pesky fatal errors?

Reply:
I'd say those are unlikely to be caused by any particular file and so it'd be fine to let those bubble up without getting in the way (and causing allocations etc)? Would be happy to include them if you prefer though.

Reply:
Sorry, I meant should we avoid trying to handle them? e.g should we use `NonFatal`?

Anyways we can always forward-fix.